chart/values.yaml

```diff
@@ -1,10 +1,6 @@
-## @section Keptn
```

Comment:
we have a follow up issue to solve this and it needs to be solved altogether in one go so that the scripts still work.
This is the follow-up issue: #2196

Reply:
fair 🐱